src/bodywork/cli/cli.py

```diff
@@ -180,11 +180,11 @@ def _debug(seconds: int = Argument(600)) -> None:
 def _create_deployment(
     git_url: str = Argument(...),
     git_branch: str = Option("", "--branch"),
-    asynchronous: bool = Option(False, "--async"),
-    asynchronous_job_name: str = Option("", "--async-job-name"),
+    asynchronous: bool = Option(False, "--async", hidden=True),
+    asynchronous_job_name: str = Option("", "--async-job-name", hidden=True),
     ssh_key_path: str = Option("", "--ssh"),
     secrets_group: str = Option("", "--group", "--secrets-group"),
-    image: str = Option(None, "--bodywork-image"),
+    image: str = Option(None, "--bodywork-image", hidden=True),
```

Comment:
I don't think we should hide the option to use a custom image. This is a useful feature and it's straightforward so I don't think it will confuse anyone. We also mention that you can override the image in the docs so we should keep the option there for people to see.